lg_stats/msg/Session.msg

```diff
@@ -0,0 +1,5 @@
+string mode
```

Comment:
Stats -> Event rename ok.
I'll look into extending the message by the JSON field which will go to telegraf. This makes sense.
Need to play with telegraf+influxdb now ( @ezekiel - I would appreciate the influxdb access details please).
Thanks the the feedback @wzin